src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx

```diff
@@ -9,15 +9,17 @@ import {
   getStatusContents,
   programAreaMap,
 } from '../../utils/appeals-v2-helpers';
+import { buildDateFormatter } from '../../utils/helpers';
+import ClaimCard from '../ClaimCard';
 
 const capitalizeWord = word => {
   const capFirstLetter = word[0].toUpperCase();
   return `${capFirstLetter}${word.slice(1)}`;
 };
 
-// This component is also used by the personalization application, which will pass the external flag.
+const formatDate = buildDateFormatter('MMMM d, yyyy');
 
-export default function AppealListItemV3({ appeal, name, external = false }) {
```

Comment:
Removing the `external` prop because it was intended to be used by the Dashboard, but they have their own component and are not importing from the `claims-status` folder